### ISSUE TITLE
Consider fewer couchdb images (i.e. get on the same page or elaborate in documentation/comments)

### ISSUE THREAD

**Issue by bean5** [apache/couchdb-docker#176]
|Project|Referencing URL|Suggested Docker Container|Reason|Pitfalls|
|----|----|----|----|----|
|Official couchdb documentation|https://docs.couchdb.org/en/2.2.0/install/docker.html|https://hub.docker.com/r/apache/couchdb| "convenience binary" | Documentation is out of date listing 2.1.0 as latest|
|fauxton (this project)|https://github.com/apache/couchdb-fauxton/blob/master/.env#L5|https://hub.docker.com/r/ibmcom/couchdb3|"Used to provide a CouchDB 2 / 3 build matrix in .travis.yml"|No mention as to why apache's convenience package doesn't work.|
|Docker hub|https://hub.docker.com/_/couchdb|https://hub.docker.com/_/couchdb|Official|Isn't enough (which is why the others exist)|

This seems unexpected to me. And causes various questions to surface such as:

* Is the official package not convenient or flexible enough?
* Why doesn't the apache one work for this project?
* This project itself is housed under the apache organization, so why use IBMs container?
* CouchDB isn't nascent anymore, but why is setting up fauxton and couchdb still messy? Standing up a docker instance is usually straight-forward especially for CouchDB after this many years, right?

**Comment by bean5** [comment]
Oh!  Thanks for all that! It was very instructive. You are spot-on that I was peeking into things and by not realizing it I got majorly confused. _I did not realize fauxton was built-in._

I've been using a container I built myself, but wanted something more mainstream. My container was built with Ubuntu 15.10 and couchdb 1.5.0. I knew if I was going to start using CouchDB full-time, I'd have to overhaul or replace. I was hoping to peek into major projects and gain some insight as to which containers would be most mainstream and trust-worthy. This one was the one that threw me off. While your selenium use was obviously top-notch, I was baffled at why the IBM container was being used instead. Your history lesson explains how it came to be and it makes sense. I'm glad you guys are careful on the legal front. Better safe than sorry.

Thank you for understanding, for being quick to close tickets I made in error, and for the lesson. I'm actually going to bookmark this page for later use.
